external/dir.proj

```diff
@@ -15,6 +15,8 @@
     <Project Include="portable\portable.depproj" />
     <Project Include="ilasm/ilasm.depproj" />
     <Project Include="docs/docs.depproj" />
+    <Project Include="winrt/winrt.depproj" />
```

Comment:
For me it was to ensure that all dependencies were being synced. This may no longer be necessary but might be something we want to explicitly condition here so that one project contains all the dependency projects.